CHANGELOG.md

```diff
@@ -2,6 +2,51 @@
 Important changes to Oh My Fish are recorded here for each release.
 
 
+## [7] - 2020-04-10
+
+Another release that took long but is finally out! v7 supports Fish 3.0 and 3.1 and brings a few improvements and bug fixes.
+
+### Improvements
+
+- Plugin and theme templates improvements. [#596] [#691] [#709]
+- Support newer versions of Fish Shell. [#609] [#618] [#644] [#710] [#711] [#740]
+- Improvements to the require mechanism. [#653] [#666] [#713]
+- Speed up Git interoperability/support. [#706]
+
+### Bugs fixed
+
+- Installation fixes and improvements. [#577] [#603] [#609] [#738] [#742]
+- Documentation fixes and improvements. [#570] [#640] [#662] [#694] [#696] [#699] [#708] [#709] [#724] [#737] [#739]
+
+### Contributors
+
+Thanks to the people who contributed to this release.
+
+- [Adam Edelstein](Adam.Edelstein@tastytrade.com)
+- [Anton-Latukha](anton.latukha@gmail.com)
+- [Benedict Lee](benedictleejh@gmail.com)
+- [Dalton Duvio](thebigdalt@gmail.com)
+- [Édouard Lopez](edouard-lopez@users.noreply.github.com)
+- [Fabian Homborg](FHomborg@gmail.com)
+- [Frederick Henderson](frederickjh@users.noreply.github.com)
+- [Gabriel César](gcal1@live.com)
+- [Jan T. Sott](jan@idleberg.com)
+- [Joel Burget](joelburget@gmail.com)
+- [Justin Hileman](mailto:github@0x7f.us)
```

Comment:
should all the rest have a `mailto:` or this one not?

Reply:
Good one too, Justin! Thanks for reviewing! Fixed.

I went over the list of contributors to this release and fixed it as well. Thanks for triggering that.

One last review?

Thanks in advance!